### DISCUSSION TITLE
how to connect looker studio

### DISCUSSION THREAD

**Discussion by aniljha-1245** [hasura/graphql-engine#9892] [Question]
to analyze data in looker studio

**Comment by martin-hasura** [comment]
As far as I know the only way to connect to external APIs using Looker is via custom connectors.

There's a guide over here for creating a connector to an endpoint:
https://medium.com/@kaushal.sonii/creating-a-custom-data-connector-for-google-looker-studio-1f816b94be43

The fundamentals for GraphQL would be the same, you're just using POST requests and passing your query as part of the body of the request. The return will be the JSON you specified as part of the GraphQL query.
